inc/Assert.h

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2017-2018 United States Government as represented by the
```

Comment:
I think there was a problem with the merge- you still have the Assert.h in this request.